Fix rule for bbb.org

muodov opened the pull request:
Task/Issue URL: https://app.asana.com/0/1209482582403805/1209556962851441

noisysocks reviewed (commented):
Something might be wrong here. When I browse to bbb.org I see a cookie popup. If I then refresh then it is handled successfully. 

There are also failing tests.

```
% npx playwright test tests/bbb.spec.ts
…
  2 failed
    [webkit] › playwright/runner.ts:48:13 › bbb.org › www.bbb.org/ .NA optOut  ─────────────────────
    [chrome] › playwright/runner.ts:48:13 › bbb.org › www.bbb.org/ .NA optOut  ─────────────────────
  2 skipped
  4 passed (17.6s)

muodov commented:
@noisysocks  I assume you were referring to this "after" popup. I changed the rule to hide it as well.
![image](https://github.com/user-attachments/assets/5eaf363d-6a7d-4e24-9920-6da7c660e218)
I also added a wait to make the test more stable, although I wonder if there's a less hacky approach.

noisysocks commented:
Are we seeing totally different banners? Running this branch and with my VPN set to San Jose USA I see this unhandled banner:

<img width="1840" alt="Screenshot 2025-03-14 at 15 27 35" src="https://github.com/user-attachments/assets/db305a13-ede9-450f-a259-80f902a56b05" />

muodov commented:
@noisysocks yeah, they have changed the layout. I updated the rule
